### PR TITLE
Remove Polyfills section from sidebar

### PR DESCRIPTION
**Pull request by JoelMarcey**
Some of these will be in basics, guides and apis instead. One less layer
of confusion.

> Note: APIs are not totally alphabetical any longer -- but neither were
Polyfills. We can fix that in `extractDocs.js` maybe. But not worth doing
in this pull request, imho.

Test Plan:

http://localhost:8079/react-native/docs/network.html

screencapture-localhost-8079-react-native-docs-network-html-1466556758780

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
By analyzing the blame information on this pull request, we identified @caabernathy and @sahrens to be potential reviewers.

File: docs/Images.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Images
 layout: docs
 category: Guides
 permalink: docs/images.html
-next: gesture-responder-system
+next: colors
```
Comment (hramos) [inline docs/Images.md]: I also changed this 'next' link in #8299 (to handling-touches), and we should coordinate to make sure these two PRs do not conflict.

I propose that Style.md be updated to point to `colors` in its next link, and Colors.md be updated to point to 'images', resulting in:

* Style
* Colors
* Images
* Handling Touches

What do you think?

File: website/server/extractDocs.js [hunk 2]
```diff
@@ -151,6 +151,7 @@ function shouldDisplayInSidebar(componentName) {
 }
 
 function getNextComponent(idx) {
+  console.log(all[idx]);
```
Comment (hramos) [inline website/server/extractDocs.js]: This can be removed.

**Comment by hramos** [comment]
Looks good, lets sync tomorrow on how to handle the links between these and the new touches guide.

File: docs/Images.md (hunk 1 above)
Comment (JoelMarcey) [inline docs/Images.md]: Sure. That's cool with me.

File: website/server/extractDocs.js (hunk 2 above)
Comment (JoelMarcey) [inline website/server/extractDocs.js]: Ah - nice catch. This was a debugging statement for me.

**Comment by JoelMarcey** [comment]
@hramos Made your suggested changes. I don't think we should be conflicting now with #8299, right?

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/294430344278750/int_phab) to review.

**Pull request closed without merging**